### PR TITLE
Refine AI settings

### LINKED ISSUE #14201: [Theia AI] Review AI settings (name, order, descriptions)

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #14201

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/ai-core/src/browser/ai-core-preferences.ts [hunk 1]
```diff
@@ -19,14 +19,14 @@ import { PreferenceProxyFactory } from '@theia/core/lib/browser/preferences/inje
...
-export const PREFERENCE_NAME_PROMPT_TEMPLATES = 'ai-features.templates.templates-folder';
+export const PREFERENCE_NAME_ENABLE_EXPERIMENTAL = 'ai-features.AiEnable.enableAI';
+export const PREFERENCE_NAME_PROMPT_TEMPLATES = 'ai-features.promptTemplates.promptTemplatesFolder';
 
 export const aiCorePreferenceSchema: PreferenceSchema = {
     type: 'object',
     properties: {
         [PREFERENCE_NAME_ENABLE_EXPERIMENTAL]: {
-            title: AI_CORE_PREFERENCES_TITLE,
+            title: 'AI_CORE_PREFERENCES_TITLE',
```
Comment (sdirix) [inline packages/ai-core/src/browser/ai-core-preferences.ts:29]: This looks like an accidental change

File: packages/ai-ollama/src/browser/ollama-preferences.ts [hunk 2]
```diff
@@ -17,16 +17,16 @@
...
+export const MODELS_PREF = 'ai-features.ollama.ollamaModels';
 
 export const OllamaPreferencesSchema: PreferenceSchema = {
     type: 'object',
     properties: {
         [HOST_PREF]: {
             type: 'string',
             title: AI_CORE_PREFERENCES_TITLE,
-            description: 'Ollama Host',
+            description: '',
```
Comment (sdirix) [inline packages/ai-ollama/src/browser/ollama-preferences.ts:29]: Should we add a description or remove the attribute?

**Review by sdirix — APPROVED** [review]
Looks much better in the preferences to me :+1:

**Comment by JonasHelming** [comment]
@planger I would need your approval as well, please

**Review by planger — APPROVED** [review]
Thanks, looks good to me!

**Pull request merged**